util/dvsim/dvsim.py

```diff
@@ -310,6 +310,12 @@ def parse_args():
                               'of categories from: {}.'.format(
                                   ', '.join(_LIST_CATEGORIES))))
 
+    parser.add_argument("--interactive",
```

Comment:
Could you please move this to the `disg` group and move it closer to `--gui`? It will also be good to clarify that this is different from GUI.